### DISCUSSION TITLE
Flipnote like animation.

### DISCUSSION THREAD

**Discussion by VeiloPot** [keiwando/toonsquid#590] [Show and tell]
I dont really talk here a lot, but, last year i made a fan animation of the June Archive series and trying to mimic the flipnote style animation. Some parts were too smooth though.

(Also warning it contains a bit of flashing images or colors)https://github.com/keiwando/toonsquid/assets/135170331/62d67bf0-551b-442a-b4b6-e2a0902c0239

**Comment by budaloops** [comment]
I love it. How did you achieve the cross-hatch texture in the shaded areas?

**Reply by VeiloPot** [reply]
Sorry for the late response but i just used the dither vertical and horizontal brushes to make the shades or other dither brushes that is in the pixel art brush folder.

**Comment by core999** [comment]
as someone who also likes flipnote animations I'm also curious of your brush set up. Do you also just set your canvas lower to match the DS resolution?

**Reply by VeiloPot** [reply]
Yes actually! I used 512 x 384 to make the pixel brushes a little more noticeable and seeable. And then i edited the animation video quality to a LITTLE low. :D
